consensus/checks.go

```diff
@@ -164,7 +174,7 @@ func (consensus *Consensus) onPreparedSanityChecks(
 			Msg("[OnPrepared] BlockHash not match")
 		return false
 	}
-	if consensus.current.Mode() == Normal {
+	if consensus.current.Mode() == Normal || consensus.current.Mode() == Syncing {
```

Comment:
This bring back the peckshield audit fix. It wasn't this which caused the signing issue.